### PR TITLE
[11.0][MIG] partner_employee_quantity: Migration to 11.0

### PR DESCRIPTION
**Pull request by pedrobaeza**
* Standard procedure
* Adapt dependencies
* Adapt menu
* Improve view layout
* README by fragments
* Rise to mature status with maintainer myself

@Tecnativa

### CODE REVIEW

File: partner_employee_quantity/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    employee_quantity = fields.Integer()
```
Comment (nikul-serpentcs) [inline partner_employee_quantity/models/res_partner.py:12]: @pedrobaeza IMO here need  `string="Employee quantity"`.

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: partner_employee_quantity/models/res_partner.py (hunk 1 above)
Comment (pedrobaeza) [reply partner_employee_quantity/models/res_partner.py:12]: That string is automatically got from the field name. In this case, it's "Employee Quantity", which is good enough.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Review by chienandalu — APPROVED** [review]
Tested on runbot. Looks good :+1:

**Pull request merged**